### DISCUSSION TITLE
Should all Keycloak issues be created here or in the respective git repo?

### DISCUSSION THREAD

**Discussion by Fydon** [keycloak/keycloak#8755] [Q&A] [answered]
With the migration from Jira to here, the closed comment says that we should create all issues here, e.g. [KEYCLOAK-13841](https://issues.redhat.com/browse/KEYCLOAK-13841). I just want to clarify if the issue is with keycloak-nodejs-connect, should I create it here or in the [keycloak-nodejs-connect](https://github.com/keycloak/keycloak-nodejs-connect) repo?

**Comment by stianst** [comment, marked as answer]
To make it a bit simpler the message in JIRA just says the main repo, and I would just move it if needed, but if you create it in the correct repo in the first place that'd be great
